lnrpc/lightning.proto

```diff
@@ -1105,6 +1105,11 @@ message EstimateFeeRequest {
 
     // Whether unconfirmed outputs should be used as inputs for the transaction.
     bool spend_unconfirmed = 4;
+
+    // CoinSelectionStrategy is the strategy to use for selecting coins
+    // to send bitcoin on-chain to multiple addresses for getting the fee
+    // estimates.
+    string coin_selection_strategy = 5;
```

Comment:
This should not be a string on the RPC level but an enum.
Then we can have the following values:

```
enum CoinSelectionStrategy {
    STRATEGY_USE_GLOBAL_CONFIG = 0; // Use whatever is configured in lnd.conf.
    STRATEGY_LARGEST = 1;
    STRATEGY_RANDOM = 2;
}
```

Then on the CLI we can have a string value that is parsed into the RPC enum values. The string values can then be `"global-config", "largest", "random"` with `"global-config"` being the default.

Reply:
This new enum should be defined in `lightning.proto` and re-used in all other proto files (by importing `lightning.proto`, if not already done).